### PR TITLE
Fhir 40566 - add event and FHIR version support

### PR DESCRIPTION
**Pull request by bvdh**

### CODE REVIEW

File: input/pagecontent/2-3-Events.md [hunk 1]
```diff
@@ -28,27 +28,27 @@ The event name defines the event. Most FHIRcast events conform to an extensible
...
 
 ```ebnf
-EventName ::= (FHIRresource | '*') ('-') ( 'open' | 'close' | 'update' | 'select' | '*' )
+EventName ::= (FHIRresource | '*') ('-') ( 'open' | 'close' | 'update' | 'select' | '*' ) (('.v') Version)?
 ```
 
 {% include img.html img="EventName.png" caption="Figure: Event-name specification" %}
 
-The `FHIRresource` indicates the focus of the event; the `suffix` defines the type of event.
+The `FHIRresource` indicates the focus of the event. The next part defines the type of event. The optional `Version` field allows specifying the version of the event. When this field is omitted, it refers to the most recent version of the event supported by the Hub. Hubs are RECOMMENDED to at least support the most recent version of the events defined in the [standard event catalog](3_Events.html).
```
[the remark this answers is not in the document]
Comment (EricOnFHIR) [reply input/pagecontent/2-3-Events.md:36]: When the `Version` field is present it SHALL be a code from the [FHIR-version](https://hl7.org/fhir/valueset-FHIR-version.html) value set.

File: input/pagecontent/3-2-1-SyncError.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ### Event-name: SyncError
 
-eventMaturity | [2 - Tested](3-1-2-eventmaturitymodel.html)
+| EventMaturity | [2 - Tested](3-1-2-eventmaturitymodel.html) |
+| Version | 1.1
```
Comment (EricOnFHIR) [inline input/pagecontent/3-2-1-SyncError.md:4]: I would like to understand the purpose of the `Version` field

**Review by EricOnFHIR — CHANGES_REQUESTED** [review]

File: input/pagecontent/2-3-Events.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bvdh) [reply input/pagecontent/2-3-Events.md:36]: Agreed on the first comment. The second one relates to the FHIR-version which is a different topic that is addressed in Conformance

**Comment by bvdh** [comment]
replaced by https://github.com/HL7/fhircast-docs/pull/585 - to facilitate easier merging

**Pull request closed without merging**